change/@fluentui-web-components-b79350c6-45b4-4047-b3cb-bdd2f3913561.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31069&commitId=c19169fa4e43023612d4a884414bc0dd32128d16&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps.normal.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>